matrix-sdk/src/main/java/org/matrix/androidsdk/rest/api/PushRulesApi.java

```diff
@@ -26,18 +27,20 @@
 import retrofit2.http.PUT;
 import retrofit2.http.Path;
 
-public interface BingRulesApi {
+public interface PushRulesApi {
 
-    // TODO Add Javadoc
+    /**
+     * Get all push rules
+     */
     @GET("pushrules/")
-    Call<BingRulesResponse> getAllBingRules();
+    Call<PushRulesResponse> getAllRules();
```

Comment:
I would rename it getAllPushRules()

Reply:
Other methods are like this (ex: `deleteRule()`) so I think it's more coherent.